test/unit/modules/compiler/parser.spec.ts

```diff
@@ -767,7 +767,7 @@ describe('parser', () => {
     expect(ast.directives[0].arg).toBe('field1')
   })
 
-  it('attribute with modifiered directive', () => {
+  it('attribute with modified directive', () => {
```

Comment:
I don't think it's a typo here. By "modifiered" it means the directive is used with modifiers(`.on` and `.off`).